mem0/configs/vector_stores/pgvector.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+from pydantic import BaseModel, Field, model_validator
+
+class PGVectorConfig(BaseModel):
+
+    dbname: str = Field("postgres", description="Default name for the database")
+    collection_name: str = Field("mem0", description="Default name for the collection")
+    embedding_model_dims: Optional[int] = Field(1536, description="Dimensions of the embedding model")
+    user: Optional[str] = Field(None, description="Database user")
+    password: Optional[str] = Field(None, description="Database password")
+    host: Optional[str] = Field("127.0.0.1", description="Database host. Default is localhost")
```

Comment:
We should not set the default value for embedding_model_dims, host and port. User should specify that.

Reply:
We need to set default embedding_model_dims because our default embedder is openai so it's dimension needs to be set.

I'll make mandatory for user to enter host and port.